benchmark/src/main/scala/keycloak/scenario/KeycloakScenarioBuilder.scala

```diff
@@ -314,6 +338,17 @@ class KeycloakScenarioBuilder {
         .body(StringBody("""{"id":"${createdRealmId}","realm":"${createdRealmId}","enabled": true}"""))
         .check(status.is(201))
         .check(header("Location").notNull.saveAs("realmLocation")))
```

Comment:
given we are not using the `realmLocation` downstream, do we need to persist this into the session ?

Reply:
@kami619 not really - it was here to keep the code stream similar, potentially making maintenance easier.
I can remove this attribute `saveAs` if you think that it's better not to store it, no pb.

Reply:
in the other methods, those location headers are consumed in, for example, `deleteClients` or `deleteClientScopes` Delete api's, here we followed a different way of deleting the Realm, hence I pointed out if we wanted to avoid saving this to session. I would rather delete it and keep the session memory lean as much as possible. Unless, you have an idea to use the Location header in any of the downstream methods for this entity.

Reply:
Done